src/main/java/it/mulders/mcs/cli/Cli.java

```diff
@@ -31,68 +35,66 @@ public class Cli {
     )
     private boolean usageHelpRequested;
 
-    public SearchCommand createSearchCommand() {
-        return new SearchCommand();
-    }
-
     public ClassSearchCommand createClassSearchCommand() {
         return new ClassSearchCommand();
     }
 
-    @CommandLine.Command(
-            name = "search",
-            description = "Search artifacts in Maven Central by coordinates",
-            usageHelpAutoWidth = true
+    @CommandLine.Parameters(
+        arity = "0..n",
```

Comment:
Needed to change from `"1..n"` to `"0..n"` to get `ClassSearch` subcommand to work.